src/app/components/theme/demos/ScoreCardDemo.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Hero, InfoListItem, ScoreCard } from '@pxblue/react-components';
+import { Temp } from '@pxblue/icons-mui';
+import { List, ListItem, ListItemIcon, ListItemText } from '@material-ui/core';
+import { CloudDone, Info, Notifications } from '@material-ui/icons';
+
+/*
+ * cannot iterate through variations due to the way MUI implemented text fields
+ */
+
+export const ScoreCardDemo: JSX.Element = (
+    <ScoreCard
```

Comment:
The color on these are busted in dark theme....is that fixed in the component library and just not published int he beta yet, or do we need to fix it?

Reply:
I did update it on my end, maybe I didn't git push yet
![image](https://user-images.githubusercontent.com/8997218/111679271-83a17300-87f7-11eb-90b6-244fa099cca7.png)